library.rb

```diff
@@ -0,0 +1,218 @@
+require "date"
+
+class Library
+  attr_reader :books, :authors
```

Comment:
add readers and orders

Reply:
done [here](https://github.com/ArtemVlasenko/Library/pull/1/commits/feffaa41e6007f75d1e75332be8f44f4148d5fc3)